placement-rules-in-sql.md

```diff
@@ -0,0 +1,154 @@
+
+The default configuration of [`max-replicas`](/pd-configuration-file.md#max-replicas) is `3`. To increase this for a specific set of tables, you can use a placement policy as follows:
+
+```sql
+CREATE PLACEMENT POLICY fivereplicas FOLLOWERS=4;
+CREATE TABLE t1 (a INT) PLACEMENT POLICY=fivereplicas;
+```
+
+Note that the PD configuration includes the leader and follower count, thus 4 followers + 1 leader equals five replicas in total.
+
+To expand on this example, the placement for the followers can also be described using the `PRIMARY_REGION` and `REGIONS` placement options:
+
+```sql
+CREATE PLACEMENT POLICY eastandwest PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-east-2,us-west-1" SCHEDULE="MAJORITY_IN_PRIMARY" FOLLOWERS=4;
+CREATE TABLE t1 (a INT) PLACEMENT POLICY=eastandwest;
+```
+
+The `SCHEDULE` instructs TiDB how to balance the followers. The default schedule of `EVEN` ensures a balance of followers in all regions. The schedule `MAJORITY_IN_PRIMARY` can be used to ensure that enough followers are placed in the primary region (`us-east-1`) that quorum can be achieved. This helps provide lower latency transactions at the expense of availability should the primary region completely fail.
+
+### Assigning placement to a partitioned table
+
+> **Note:**
+>
+> This example makes use of list partitioning, which is currently an experimental feature. Partitioned tables also require the `PRIMARY KEY` to be included in all columns in the table's partitioning function.
+
+As well as assigning to tables, placement options can also be assigned to table partitions. For example:
+
+```sql
+CREATE PLACEMENT POLICY europe PRIMARY_REGION="eu-central-1" REGIONS="eu-central-1,eu-west-1";
+CREATE PLACEMENT POLICY northamerica PRIMARY_REGION="us-east-1" REGIONS="us-east-1";
+
+SET tidb_enable_list_partition = 1;
+CREATE TABLE t1 (
+  country VARCHAR(10) NOT NULL,
+  userdata VARCHAR(100) NOT NULL
+) PARTITION BY LIST COLUMNS (country) (
+  PARTITION pEurope VALUES IN ('DE', 'FR', 'GB') PLACEMENT POLICY=europe,
+  PARTITION pNorthAmerica VALUES IN ('US', 'CA', 'MX') PLACEMENT POLICY=northamerica
+);
+```
+
+### Setting the default placement for a schema
+
+Default placement options can be directly attached to a database schema. This works similar to setting the default character set or collation for a schema, in that it will be used when no other placement options are specified. For example:
+
+```sql
+CREATE TABLE t1 (a INT); -- the table is created with no placement options
+ALTER DATABASE test FOLLOWERS=4; -- this changes the default, and does not apply to the existing table t1;
+CREATE TABLE t2 (a INT); -- the placement of FOLLOWERS=4 will be used
+CREATE TABLE t3 (a INT) PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-east-2"; -- FOLLOWERS=4 does not apply as placement is specified.
+ALTER DATABASE test FOLLOWERS=2; -- this does not apply to existing tables
+CREATE TABLE t4 (a INT); -- the table is created with FOLLOWERS=2;
```

Comment:
```suggestion
CREATE TABLE t1 (a INT);  -- Creates a table t1 with no placement options.

ALTER DATABASE test FOLLOWERS=4;  -- Changes the default placement option, and does not apply to the existing table t1.

CREATE TABLE t2 (a INT);  -- Creates a table t2 with the default placement of FOLLOWERS=4.

CREATE TABLE t3 (a INT) PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-east-2";  -- Creates a table t3 without the default FOLLOWERS=4 placement, because this statement has specified another placement.

ALTER DATABASE test FOLLOWERS=2;  -- Changes the default placement, and does not apply to existing tables.

CREATE TABLE t4 (a INT);  -- Creates a table t4 with the default FOLLOWERS=2 option.
```